build.py

```diff
@@ -9,17 +9,29 @@
 def create_parser():
     parser = argparse.ArgumentParser(description="Build PEP documents")
     # alternative builders:
-    parser.add_argument("-l", "--check-links", action="store_true")
-    parser.add_argument("-f", "--build-files", action="store_true")
-    parser.add_argument("-d", "--build-dirs", action="store_true")
+    parser.add_argument("-l", "--check-links", action="store_true",
+                        help='Checks validity of links within PEP sources. '
+                             'Cannot be used with "-f" or "-d".')
+    parser.add_argument("-f", "--build-files", action="store_true",
+                        help='Renders PEPs to "pep-XXXX.html" files (Default). '
+                             'Cannot be used with "-d" or "-l".')
+    parser.add_argument("-d", "--build-dirs", action="store_true",
+                        help='Renders PEPs to "index.html" files within "pep-XXXX" directories. '
+                             'Cannot be used with "-f" or "-l".')
 
     # flags / options
-    parser.add_argument("-w", "--fail-on-warning", action="store_true")
-    parser.add_argument("-n", "--nitpicky", action="store_true")
-    parser.add_argument("-j", "--jobs", type=int, default=1)
+    parser.add_argument("-w", "--fail-on-warning", action="store_true",
+                        help="Fails Sphinx on warnings.")
+    parser.add_argument("-n", "--nitpicky", action="store_true",
+                        help="Runs Sphinx in 'nitpicky' mode, "
+                             "warning on every missing reference target.")
+    parser.add_argument("-j", "--jobs", type=int, default=1,
+                        help="How many parallel jobs to run (if supported). "
+                             "Integer, default 1.")
 
     # extra build steps
-    parser.add_argument("-i", "--index-file", action="store_true")  # for PEP 0
+    parser.add_argument("-i", "--index-file", action="store_true",  # for PEP 0
+                        help="Copies PEP 0 to a base index file.")
```

Comment:
```suggestion
                        help="Copy PEP 0 to a base index file.")
```

Its still a little unclear to me what this does, or when it is needed. Could you describe it in a bit more detail?

Reply:
I removed the argument and now unconditionally run it.